container-toolkit/release-notes.md

```diff
@@ -10,6 +10,49 @@ This document describes the new features, improvements, fixed and known issues f
 
 ______________________________________________________________________
 
+## NVIDIA Container Toolkit 1.15.0
+
+This release of the NVIDIA Container Toolkit `v1.15.0` is a major feature release.
+
+The following packages are included:
+
+- `nvidia-container-toolkit 1.15.0`
+- `libnvidia-container-tools 1.15.0`
+- `libnvidia-container1 1.15.0`
+
+   ```{note}
+   The `v1.14.0` release was the last release to include the `nvidia-container-runtime`
+   and `nvidia-docker2` packages.
+   All required functionality is included in the `nvidia-container-toolkit` package.
+   This toolkit package includes a utility to configure the Docker daemon to use the NVIDIA Container Runtime.
+   ```
+
+The following `container-toolkit` containers are included:
+
+- `nvcr.io/nvidia/k8s/container-toolkit:v1.15.0-ubi8`
+- `nvcr.io/nvidia/k8s/container-toolkit:v1.15.0-ubuntu20.04` (also as `nvcr.io/nvidia/k8s/container-toolkit:v1.15.0`)
+
+### Packaging Changes
```

Comment:
Teensy nit: Do you mind adding a blank line after this heading?

Reply:
Done in #43.